frontend/src/Explorer/View/Transaction.purs

```diff
@@ -86,12 +86,6 @@ summaryItems (CTxSummary txSummary) lang =
                 in bHeight <> " (" <> bTime <> ")"
       , currency: Nothing
       }
-    , { label: translate (I18nL.tx <<< I18nL.txRelayed) lang
```

Comment:
@ksaric is relayedBy the same as Leader ? Because I think it is. Leader publishes a block. 

If this is the case then this commit https://github.com/input-output-hk/cardano-sl-explorer/pull/107#event-1078176190 in fact adds some data to relayed by - thus it doesn't have to be removed.

@ksaric right ?